atlas_densities/densities/mtype_densities_from_map/create.py

```diff
@@ -148,7 +148,12 @@ def _create_densities_for_metype(metype: str) -> Optional[Tuple[str, str]]:
 
     metadata = {
         "synapse_class": synapse_class,
-        "density_files": output_legend,
+        "metype2file": output_legend,
+        "file2metype": {
```

Comment:
I don't think we need this; reorganizing the dictionary hierarchies isn't too complicated, and the MType/Etype/file is what we have used in other NEXUS places, so I think we should only keep that standard.

Reply:
There is no consumer of this dictionary other than the Atlas pipeline, and it does not need any reorganization if provided in this format.

Reply:
I'm not a big fan of changing the layout, since it's generally what we've done in other config files, but if is causing this many problems, do use only the `file2metype` layout so this can be merged.

Reply:
Yes, that works.

Reply:
> Yes, that works.

I'm sorry, I've lost the context on this.  Which "works", the reorganized or non reorganized one?

Reply:
I used the `metype2file` and reorganized [myself](https://bbpgitlab.epfl.ch/dke/apps/blue_brain_atlas_nexus_push/-/blob/96728ad772f9dc39dee60d37b7671e429942de45/bba_data_push/push_nrrd_volumetricdatalayer.py#L118).